dynamics/src/ParametricMesh.cpp

```diff
@@ -189,6 +189,80 @@ void ParametricMesh::readmesh(std::string fname)
     }
 }
```

Comment:
It would be useful to add some docstrings/function documentation.

Reply:
Copied the Doxygen documentation from the header file.